### PR TITLE
Added the Meeting Scheduler Card

### PR DESCRIPTION
**Pull request by Randula98**
image

### CODE REVIEW

File: src/pages/dashboard/index.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { applications as getApplications } from "../../store/application-store/a
 import { getAllUsers } from "../../store/user-store/userActions";
 import { getEvents } from "../../store/event-store/eventActions";
 import { getInquiries } from "../../store/inquiry-store/inquiryAction";
+import MeetingScheduler from "./meeting_scheduler";
```
Comment (senuravihanjayadeva) [inline src/pages/dashboard/index.tsx:12]: Move this import after line no 8

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Review by KaveenLaksitha — COMMENTED** [review]
LGTM

File: src/pages/dashboard/index.tsx (hunk 1 above)
Comment (Randula98) [reply src/pages/dashboard/index.tsx]: image
Done..

**Review by uihalagedara — APPROVED** [review]
LGTM

**Review by ThusharaX — APPROVED** [review]
LGTM

**Pull request merged**